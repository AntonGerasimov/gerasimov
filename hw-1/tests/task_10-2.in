-2 100 23.5
