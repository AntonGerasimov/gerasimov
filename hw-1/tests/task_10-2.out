-2
23.500000
d
